modules/framework/galasa-parent/dev.galasa.framework.api.streams/src/test/java/dev/galasa/framework/api/streams/internal/routes/StreamsByNameRouteTest.java

```diff
@@ -277,4 +279,162 @@ public void testGetSTreamsByNameRouteThrowsInternalServletException() throws Exc
 
     }
 
+
+//      DELETE STREAM BY NAME TESTS     ////
+
+    @Test
+    public void testDeleteStreamsByAMtachingNameWithMultipleStreamsReturnsRequiredStreamOK() throws Exception {
```

Comment:
Typo in this function name "Mtaching". This could be slightly shortened to something like testDeleteStreamWithMultipleStreamsDeletesCorrectStream